newm-server/src/main/kotlin/io/newm/server/database/migration/V13__CreateConfig.kt

```diff
@@ -0,0 +1,19 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V13__CreateConfig : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            execInBatch(
+                listOf(
+                    "CREATE TABLE IF NOT EXISTS config (id text PRIMARY KEY, value TEXT NOT NULL)",
+                    "INSERT INTO config VALUES ('mint.price','6000000') ON CONFLICT(id) DO NOTHING"
```

Comment:
Adding the `mint.price` initial value here since this is the only value we currently have and we are creating the table on this migration. In the future, when we add new values we don't need to do a database migration. We can just pre-load the values directly into the database before merging PRs.